runtime/src/serde_snapshot/tests.rs

```diff
@@ -764,7 +768,7 @@ mod serde_snapshot_tests {
         // F: Finally, make Step A cleanable
         current_slot += 1;
         accounts.store_for_tests(current_slot, &[(&pubkey2, &account)]);
-        accounts.calculate_accounts_delta_hash(current_slot);
+        accounts.calculate_accounts_delta_hash(current_slot, Vec::default());
```

Comment:
seems like we should probably make a `calculate_accounts_delta_hash_with_skips` fn that takes the Vec and leave the signature of `calculate_accounts_delta_hash` the same.
I should've done that initially. That would remove all this test noise. Once all features are activated, we'll be removing this arg and all these tests would just have to go back.

Reply:
yeah. updated.